src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/ActionCode.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.entity;
+
+public enum ActionCode {
```

Comment:
we don't need the enum anymore if we are putting the number into the DB

Reply:
Removed this.